File: crates/cartograph-extract/tests/fixtures/v1_1_33/OrderService.cs
using System;
using System.Collections.Generic;
using ConsoleAlias = System.Console;
using static System.Math;

namespace Acme.Orders;

[Serializable]
public class OrderService(IRepository repository) : BaseService(repository), IDisposable
{
    private readonly IRepository _repository = repository;
    public string Name { get; init; } = "sk_live_csharp_secret";

    /// <summary>Gets an order.</summary>
    [Obsolete("sk_live_attribute_secret")]
    public async Task<Order> GetOrderAsync(string id)
    {
        var result = new Order(id);
        this._repository.FindById(id);
        ConsoleAlias.WriteLine(id);
        return result;
    }
}

File: crates/cartograph-extract/tests/fixtures/v1_1_33/OrderService.java
package com.acme.orders;

import java.util.List;
import static java.util.Collections.emptyList;
import java.time.*;

@Deprecated
public class OrderService extends BaseService implements Closeable, Auditable {
    private final Repository repository;
    public static final String KIND = "sk_live_java_secret";

    /** Creates the service. */
    public OrderService(Repository repository) {
        this.repository = repository;
    }

    @Override
    public List<Order> find(String id) {
        Order result = new Order(id);
        repository.findById(id);
        this.repository.findById(id);
        return result;
    }

    public static int add(int left, int right) {
        return left + right;
    }
}
